app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __repr__(self):
+        return "{" + f"Name: {self.name}," \
+                     f" Health: {self.health}," \
+                     f" Hidden: {self.hidden}" + "}"
+
+    def __init__(self, name, health=100, hidden=False):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
I think better place def __init__ at the start of class. 
We can see class variables (alive and __init__) next to each other, no need to look for what variables in this class - everything at the beginning